File: Java/CesiumLanguageWriterTests/translatedSrc/cesiumlanguagewritertests/data/EmbeddedData.java
package cesiumlanguagewritertests.data;


import agi.foundation.compatibility.*;
import agi.foundation.compatibility.Assembly;
import cesiumlanguagewritertests.*;
import java.io.InputStream;
import javax.annotation.Nonnull;
import org.junit.Assert;

@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public final class EmbeddedData {
    private EmbeddedData() {}

    @Nonnull
    public static InputStream read(@Nonnull String file) {
        InputStream stream = Assembly.getExecutingAssembly().getManifestResourceStream(EmbeddedData.class, file);
        Assert.assertNotNull(stream);
        return stream;
    }
}

File: DotNet/CesiumLanguageWriterTests/Data/EmbeddedData.cs
using System.IO;
using System.Reflection;
using JetBrains.Annotations;
using NUnit.Framework;

namespace CesiumLanguageWriterTests.Data
{
    public static class EmbeddedData
    {
        [NotNull]
        public static Stream Read([NotNull] string file)
        {
            var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream(typeof(EmbeddedData), file);
            Assert.IsNotNull(stream);
            return stream;
        }
    }
}
